src/main/java/com/google/cloud/tools/gradle/appengine/standard/AppEngineStandardPlugin.java

```diff
@@ -118,7 +123,12 @@ private void configureExtensions() {
                   .resolve("WEB-INF")
                   .resolve("appengine-web.xml")
                   .toFile();
-          deploy.setDeployTargetResolver(new StandardDeployTargetResolver(appengineWebXml));
+
+          // configure the deploy extensions's project/version parameters
+          StandardDeployTargetResolver resolver =
```

Comment:
Change here: deploy now computers at configuration time the projectId and version.